vector/src/main/java/im/vector/app/features/login/LoginActivity.kt

```diff
@@ -218,10 +218,7 @@ open class LoginActivity : VectorBaseActivity<ActivityLoginBinding>(), UnlockedA
                 // change the screen name
                 analyticsScreenName = MobileScreen.ScreenName.Register
             }
-            val intent = HomeActivity.newIntent(
-                    this,
-                    accountCreation = loginViewState.signMode == SignMode.SignUp
-            )
+            val intent = HomeActivity.newIntent(this)
```

Comment:
Just to be sure to understand, why don't we pass a value for `authenticationDescription` param here?
Applicable to all `HomeActivity.newIntent` invocation.

Reply:
it's because the sign up tracking has only been implemented for the FTUE flow, the previous `accountCreation` was unused , although now I can see that develop has changed make use of the flag for secure backup/cross signing

will look to reintroduce

Reply:
https://github.com/vector-im/element-android/pull/6042/commits/49930773e7254c23d0b01b26bfb59154d43c2df8